pkg/commands/run.go

```diff
@@ -157,7 +157,7 @@ func (c *runCommand) persistentPreRunE(cmd *cobra.Command, _ []string) error {
 
 	if c.cfg.Run.Concurrency == 0 {
 		// Automatically set GOMAXPROCS to match Linux container CPU quota.
-		_, _ = maxprocs.Set(nil)
+		_, _ = maxprocs.Set(maxprocs.Logger(c.log.Infof))
```

Comment:
why not return the error or do fallback (on error) to `else` branch?

Reply:
because the fallback is already inside the function.

Reply:
hmm, I was worried about this one

```
	maxProcs, status, err := cfg.procs(cfg.minGOMAXPROCS)
	if err != nil {
		return undoNoop, err
	}
```

Reply:
https://github.com/golangci/golangci-lint/pull/4492